artiq/examples/master/zotino.py

```diff
@@ -0,0 +1,41 @@
+from artiq.experiment import *
+
+class ZotinoTestDAC(EnvExperiment):
+    def build(self):
+        self.setattr_device("core")
+        self.setattr_device("dac_zotino")
+        self.setattr_device("latch_config")
+        self.setattr_device("clk_config")
+        self.setattr_device("ser_config")
+		
+    @kernel
+    def dir_config(self, rclk, srclk, ser, data): 
```

Comment:
I would call those consistently with ``io_config`` and with better names, e.g. ``clk``, ``ser_data``, ``latch``.